airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceSSLCertificateAcceptanceTest.java

```diff
@@ -60,11 +61,10 @@ protected void setupEnvironment(final TestDestinationEnv environment) throws Exc
     final JsonNode replicationMethod = Jsons.jsonNode(ImmutableMap.builder()
         .put("method", "Standard")
         .build());
-    final var containerOuterAddress = SshHelpers.getOuterContainerAddress(container);
-    final var containerInnerAddress = SshHelpers.getInnerContainerAddress(container);
+
     config = Jsons.jsonNode(ImmutableMap.builder()
-        .put("host", containerInnerAddress.left)
-        .put("port", containerInnerAddress.right)
+        .put("host", HostPortResolver.resolveHost(container))
```

Comment:
Is this going to return the inner address and port of a container? This was the reason tests were originally failing when run on Mac.
Can you validate that running integration test on your Mac is still passing.
For example:
```
./gradlew --info :airbyte-integrations:connectors:source-postgres:integrationTestJava --tests "io.airbyte.integrations.io.airbyte.integration_tests.sources.PostgresSourceSSLCaCertificateAcceptanceTest"
```

Reply:
Yes it passed, this is why I made the change...
<img width="1964" alt="Screen Shot 2023-05-11 at 19 34 13" src="https://github.com/airbytehq/airbyte/assets/5551758/6de87aad-4044-4f95-bbb9-c28aa69fe254">

Reply:
thanks 
I'm just making sure as I don't know dagger 😅